plugins/uploadwidget/plugin.js

```diff
@@ -251,8 +251,9 @@
 
 				loader.on( 'update', function( evt ) {
 					// Abort if widget was removed.
-					if ( !widget.wrapper || !widget.wrapper.getParent() ) {
-						if ( !editor.editable().find( '[data-cke-upload-id="' + id + '"]' ).count() ) {
+					if (!widget.wrapper || !widget.wrapper.getParent()) {
```

Comment:
Please adapt your code style to other part of CKEdtor. More information about styling you can find [here](https://github.com/ckeditor/ckeditor-idiomatic.js). It's helpful to install `githooks` which checks styling when you perform a commit. Information about githooks you can find [here](https://docs.ckeditor.com/#!/guide/dev_contributing_code)